Controller/TwigExceptionController.php

```diff
@@ -64,4 +74,33 @@ protected function findTemplate(Request $request, $statusCode, $showException)
 
         return sprintf('@Twig/Exception/%s.html.twig', $showException ? 'exception_full' : $name);
     }
+
+    /**
+     * See if a template exists using the modern Twig mechanism.
+     *
+     * This code is based on TwigBundle and should be removed when the minimum required
+     * version of Twig is >= 3.0. See src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php
+     *
+     * @param string $template
+     *
+     * @return bool
+     */
+    private function templateExists(string $template): bool
+    {
+        $template = (string) $template;
```

Comment:
looks like this is not needed anymore since we have the parameter type declaration now :)

Reply:
Oops, missed that!